.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Hi @bnjjj 
It's OK for me but there are two things to notice : 
 - you already have a .editorconf in docs/ directory and this one specify tabs instead of spaces, 
- this file also should not begin with section "root=true" because it is not the root one. 

Let me know what you think about these.

Reply:
yes you can change root attribute on the .editorconfig in docs/ I think it's a default .editorconfig coming from gohugo. But keep spaces to 4 for .ts .css and .html (I already know that not all files respect this rules but it's our bad).

Reply:
Updated. Please let me know if it's ok.

Reply:
LGTM, thanks for your contribution 🎉